test/boost/tablets_test.cc

```diff
@@ -2182,9 +2183,9 @@ SEASTAR_THREAD_TEST_CASE(test_load_balancer_shuffle_mode) {
   do_with_cql_env_thread([] (auto& e) {
     topology_builder topo(e);
 
-    auto host1 = topo.add_node(node_state::normal, 1);
-    auto host2 = topo.add_node(node_state::normal, 1);
-    topo.add_node(node_state::normal, 2);
+    auto host1 = topo.add_node(node_state::normal, 1, endpoint_dc_rack{.dc = "dc1", .rack = "r1"});
+    auto host2 = topo.add_node(node_state::normal, 1, endpoint_dc_rack{.dc = "dc1", .rack = "r2"});
+    topo.add_node(node_state::normal, 2, endpoint_dc_rack{.dc = "dc1", .rack = "r2"});
```

Comment:
Proper way to do it is to create racks using `topo` builder:

``` 
 auto host1 = topo.add_node(node_state::normal, 1);
 topo.start_new_rack();
 auto host2 = topo.add_node(node_state::normal, 1);
 topo.add_node(node_state::normal, 2);
```

or (more verbose, not for here):

```
auto rack1 = topo.rack();
auto rack2 = topo.start_new_rack();

auto host1 = topo.add_node(node_state::normal, 1, rack1);
auto host2 = topo.add_node(node_state::normal, 1, rack2);
topo.add_node(node_state::normal, 2, rack2);
```

Reply:
Same for other tests.